packages/charts-react-line/src/LineChart.js

```diff
@@ -72,13 +86,21 @@ export default class LineChart extends React.PureComponent {
 
   render() {
     const { line, props } = this;
-    const { data, margin, width, height } = props;
+    const { data, margin, strokes, width, height } = props;
+    const lines = this.props.lines.map((data, index) =>
+      <Line
+        line = {line}
+        data = {data}
+        key={index}
+        stroke={strokes[index % strokes.length]}
```

Comment:
The cycle is to cover if they do not define enough colors for the amount of lines made from a data source.